lib/css/atomic/_stacks-flex.less

```diff
@@ -196,6 +196,7 @@
 //  $   FLEX SPACING
 //      Insert spaces or gutters in between flex items
 //  ----------------------------------------------------------------------------
+// TODO: Make this play nice with spacing units… or yeet this in the age of gap
```

Comment:
Yeah, I consider this a long-term TODO. 

That said, if we wind up with `@su` only here (and maybe one other spot), it might be worth it to just hardcode the pixel values here for the sake of simplicity. No rush from me though.